.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
         rust: [stable]
+        rust-msrv: ["1.63"]
```

Comment:
we should document this in `Cargo.toml`s and in the `README.md`s

Reply:
Oh, I'd forgotten this was a Cargo.toml field now.

I've not added it to the READMEs yet, mostly because I'm not sure I like having to keep modifying so many files every time we bump it.  Especially if the Cargo.toml already contains it as metadata (which should be able to moved to the workspace Cargo.toml once the MSRV is 1.64, which in turn will probably happen when we add quic-rpc because that uses GATs)